site/src/documents/pages/blocks/FooterContentBlock.tsx

```diff
@@ -0,0 +1,138 @@
+import { PropsWithData, withPreview } from "@comet/cms-site";
+import { FooterContentBlockData } from "@src/blocks.generated";
+import { DamImageBlock } from "@src/common/blocks/DamImageBlock";
+import { LinkBlock } from "@src/common/blocks/LinkBlock";
+import { RichTextBlock } from "@src/common/blocks/RichTextBlock";
+import { Typography } from "@src/common/components/Typography";
+import * as React from "react";
+import styled from "styled-components";
+
+export const FooterContentBlock = withPreview(
+    ({ data: { text, image, linkList, copyrightNotice } }: PropsWithData<FooterContentBlockData>) => {
+        return (
+            <Root>
+                <ImageTextWrapper>
+                    <ImageWrapper>
+                        <DamImageBlock data={image} aspectRatio="1/1" objectFit="contain" />
```

Comment:
```suggestion
                        <DamImageBlock data={image} aspectRatio="1/1" style={{ object-fit="contain" }} />
```

the prop `objectFit` is deprecated by next/image, you should set the style.